chat-formatter/src/com/eternalcode/formatter/ChatSettings.java

```diff
@@ -2,7 +2,7 @@
 
 public interface ChatSettings {
 
-    boolean receiveUpdates();
+    boolean isReceiveUpdates();
```

Comment:
wcześniej dobrze było ;-;, imo

Reply:
Ale to jest boolean, przez co wcześniej nazwa nie była opisowa, gdy teraz ktoś będzie chciał użyć ChatSettings#isReceiveUpdates to od razu wiadomo że to zwraca booleana i sprawdza czy jest włączona ta opcja configu. Poprzednia wersja nie mówiła zbyt wiele

Reply:
w praktyce to w intelce obok metody masz napisane co zwraca, ale to twoja decyzja

Reply:
Intelka to nie wszystko, github ci tego nie powie. Kod powinien być czytelny i opisowy bezwzględu na ide.